test/files/Onda.jl/src/samples.jl

```diff
@@ -108,10 +107,10 @@ channel(samples::Samples, i::Integer) = channel(samples.signal, i)
 Returns the `Nanosecond` value for which `samples[TimeSpan(0, duration(samples))] == samples.data`.
 
 !!! warning
-    `duration(samples)` is not generally equivalent to `duration(samples.signal)`;
-    the former is the duration of the entire original signal in the context of its
-    parent recording, whereas the latter is the actual duration of `samples.data`
-    given `samples.signal.sample_rate`.
+`duration(samples)` is not generally equivalent to `duration(samples.signal)`;
```

Comment:
Seems not to understand note syntax.

Reply:
You'll need `[AdmonitionRule(), FootnoteRule(), MathRule(), TableRule()]`. I *think* that's everything needed, but their may be a few other small differences such as https://github.com/MichaelHatherly/CommonMark.jl/issues/6, which may be a deal breaker for this until it's resolved.

Reply:
Cool! Thanks for the help. I don't see that issue as a big problem as long as it'll be fixed eventually.